main.go

```diff
@@ -75,5 +88,8 @@ func main() {
 
 func init() {
 	flag.StringVar(&kubeconfig, "kubeconfig", "", "Path to a kubeconfig. Only required if out-of-cluster.")
-	flag.StringVar(&masterURL, "master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
+
+	flag.StringVar(&etcdURL, "etcd-core-url", "", "The address of the core etcd server. Required.")
```

Comment:
follow up issue to use cobra.  It will be easier for kube developers to follow what's happening.

Reply:
Created issues #5 and #7 to track this.